tests/unit-tests/payment-gateways/payment-gateways.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @package WooCommerce/Tests/PaymentGateways
+ */
+
+/**
+ * Class WC_Tests_Payment_Gateway.
+ */
+class WC_Tests_Payment_Gateway extends WC_Unit_Test_Case {
+
+	/**
+	 * Setup, enable payment gatewars Cash on delivery and direct bank deposit.
+	 */
+	public function setUp() {
+		parent::setUp();
+		WC()->session = null;
+		$wc_payment_gateways = WC_Payment_Gateways::instance();
+		$wc_payment_gateways->init();
+		foreach ( $wc_payment_gateways->payment_gateways() as $name => $gateway ) {
+			if ( in_array( $name, array( 'cod', 'bacs' ) ) ) {
+				$gateway->enabled = 'yes';
+			}
+		}
+	}
+
+	/**
+	 * Initialize session that some tests might have removed.
+	 */
+	public function tearDown() {
+		parent::tearDown();
+		WC()->initialize_session();
+	}
+
+	/**
+	 * Test that we can set a current gateway from session. Basic smoke test.
+	 */
+	public function test_wc_set_current_gateway_from_session() {
+		WC()->initialize_session();\
```

Comment:
It looks like there is a typo in this line with a `\` at the end of the line.

Reply:
Nice catch. Fixed now.